src/api/articleApi.ts

```diff
@@ -29,7 +30,44 @@ export async function fetchArticle(
     `${host}/article-converter/json/${context.language}/${params.articleId}?1=1${filterParam}${subjectParam}${oembedParam}${pathParam}`,
     context,
   );
-  return resolveJson(response);
+
+  const concept = await resolveJson(response);
```

Comment:
Concept i articleApi høres feil ut. Er dette kopiert fra andre plasser brukt i listing-frontend? kunne du heller gjenbrukt koden i stedet for å kopiere?